### PR TITLE
RHDEVDOCS-5141: Document modifying MachineConfigs

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **OCP version for cherry-picking**: enterprise-4.10 and later
• **JIRA issue**: [RHDEVDOCS-5141](https://issues.redhat.com/browse/RHDEVDOCS-5141)
• **Preview page**:  [Troubleshooting issues in Red Hat OpenShift GitOps](https://58636--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/troubleshooting-issues-in-gitops)
• **SME Review**: Completed by @iam-veeramalla
• **QE review**: Completed by @varshab1210
• **Peer-review**: Completed by @Srivaralakshmi

### CODE REVIEW

**Comment by ekristova** [comment]
Hello @iam-veeramalla @varshab1210 PTAL and provide feedback. Thank you!

**Comment by varshab1210** [comment]
LGTM, thanks!

**Comment by Srivaralakshmi** [comment]
@ekristova The OCP version for cherry-picking should be enterprise-4.9 and later.

@varshab1210 and @iam-veeramalla can you please confirm this?

**Comment by Srivaralakshmi** [comment]
@varshab1210 and @iam-veeramalla I see that the s [RHDEVDOCS-5141](https://issues.redhat.com//browse/RHDEVDOCS-5141) JIRA issue affects GitOps 1.7, GitOps 1.8. And that the TS section has been added only from OCP 4.11.

Should the OCP version for cherry-picking the TS section be: `enterprise-4.10` and later? That way the TS section will be in OCP 4.10 as well. WDYT?

File: cicd/gitops/troubleshooting-issues-in-GitOps.adoc [hunk 1]
```diff
@@ -1,12 +1,18 @@
 :_content-type: ASSEMBLY
 [id="troubleshooting-issues-in-GitOps"]
 
-= Troubleshooting issues in Red Hat OpenShift GitOps
+= Troubleshooting issues in {gitops-title}
```
Comment (Srivaralakshmi) [inline cicd/gitops/troubleshooting-issues-in-GitOps.adoc:4]: very good

**Review by Srivaralakshmi — APPROVED** [review]
@ekristova Splendid job! Approved : )

**Comment by Srivaralakshmi** [comment]
> @varshab1210 and @iam-veeramalla I see that the s [RHDEVDOCS-5141](https://issues.redhat.com//browse/RHDEVDOCS-5141) JIRA issue affects GitOps 1.7, GitOps 1.8. And that the TS section has been added only from OCP 4.11.
> 
> Should the OCP version for cherry-picking the TS section be: `enterprise-4.10` and later? That way the TS section will be in OCP 4.10 as well. WDYT?

@ekristova Please follow up with SMEs on this and input the CP versions in the PR description as required. Thanks!

**Comment by varshab1210** [comment]
> @varshab1210 and @iam-veeramalla I see that the s [RHDEVDOCS-5141](https://issues.redhat.com//browse/RHDEVDOCS-5141) JIRA issue affects GitOps 1.7, GitOps 1.8. And that the TS section has been added only from OCP 4.11.
> 
> Should the OCP version for cherry-picking the TS section be: `enterprise-4.10` and later? That way the TS section will be in OCP 4.10 as well. WDYT?

Sounds good

**Pull request merged**